app/components/views/ShutdownPage/ShutdownPage.jsx

```diff
@@ -0,0 +1,24 @@
+import { FormattedMessage as T } from "react-intl";
+import { useShutdown } from "./hooks";
+import { useMountEffect } from "hooks";
+import { DecredLoading } from "indicators";
+import { classNames } from "pi-ui";
+import styles from "../GetStartedPage/GetStarted.module.css";
```

Comment:
This isn't ideal, the following classes in GetStarted.module.css are shared between different files, 
`<div className={classNames(styles.pageBody, styles.getstarted)}>` we should wrap this div in a component 
by it's own, as it used in many places, then we will avoid importing this module here, I will keep it here for know and will open a separate issue regarding this reused div/classes.

Reply:
fine by me